src/main/resources/openapi/audio.yaml

```diff
@@ -17,6 +17,12 @@ paths:
       operationId: addAudioRequest
       description: Adds a user audio request for processing
       responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AddAudioResponse'
         '201':
```

Comment:
The 201 response should be removed if we want to just have the 200 response now?